server/main.go

```diff
@@ -47,24 +71,28 @@ func main() {
 	r.Run(":" + strconv.Itoa(config.Port))
 }
 
+// NewApp creates the app that holds all the functions to interact with the DB
 func NewApp() App {
 	app := App{
-		db: &inmemory.InMemory{},
+    db: &psql.DB{},
```

Comment:
remember to format your code :) (`go fmt` in the command line, but your editor should have an autoformatter)